README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# HW01 Computing for the Social Sciences 
+
+*author:Chloe Zhao* 
+  
+
+![me studying](kitty.jpg)
+
+
+1. different kinds of cute kitties 
```

Comment:
Might look better as a header and a list:

```
# Different kinds of cute kitties

+ British Shorthair 
+ Ragdoll
+ Persian Cat
```

# Different kinds of cute kitties

+ British Shorthair 
+ Ragdoll
+ Persian Cat